ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -1515,7 +1522,7 @@ def set_deviceset_count(count):
     )
 
 
-def get_storage_cluster(namespace=config.ENV_DATA["cluster_namespace"]):
+def get_storage_cluster(namespace=None):
```

Comment:
@ebondare even in the provider mode of deployment, as of now the storagecluster creation would be in the "openshift-storage" namespace. Is there a reason why we changed this line of code here?

Reply:
The reason is that when we switch from client to provider or back, the current config is where we need to get namespace from. When the namespace is defined like in line 1518, it could mean that the namespace stays openshift-storage-client, not openshift-storage, even after the context switch. I had a few test runs that failed because of it, I can try to find the links.